addon/components/sl-tooltip.js

```diff
@@ -29,6 +29,26 @@ export default Ember.Component.extend( TooltipEnabled, {
     // -------------------------------------------------------------------------
     // Observers
 
+    /**
+     * Check passed parameters on initialization
+     *
+     * @function
+     * @returns {undefined}
+     */
+    initialize: Ember.on (
+        'init',
+        function() {
+            Ember.assert(
+                'method/enablePopover() expects the parameter "title" and for it to be a string',
+                'string' === Ember.typeOf ( this.get( 'title' ) )
+            );
+            Ember.assert(
+                'method/enablePopover() expects the parameter "popover" and for it to be a string',
+                'string' === Ember.typeOf ( this.get( 'popover' ) ) || 'undefined' === Ember.typeOf ( this.get( 'popover' ) )
```

Comment:
The second condition will never be reached because the first one will catch the string of "undefined"